packages/bpk-component-slider/package.json

```diff
@@ -18,6 +18,7 @@
     "bpk-react-utils": "^2.3.6",
     "bpk-tokens": "^26.4.5",
     "prop-types": "^15.5.8",
-    "react-slider": "^0.9.0"
+    "react-slider": "^0.9.0",
+    "bpk-component-rtl-toggle": "^1.0.28"
```

Comment:
Ah - so we don't want to introduce `bpk-component-rtl-toggle` as a hard dependency of the slider - we only want to "opt in" to this functionality for the docs or the storybook. So this should be listed as a `devDependency`.